docs/_includes/chips-combobox.html

```diff
@@ -0,0 +1,396 @@
+                <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                    <use href="#icon-close-12"></use>
+                </svg>
+            </button>
+        </span>
+    </div>
+    <span class="combobox combobox--js chips-combobox__combobox">
+        <span class="combobox__control chips-combobox_combobox__control">
+            <input id="chips-combobox-1-input" role="combobox" type="text" placeholder="Add Sport" aria-haspopup="listbox" aria-owns="listbox-chips-combobox-1" />
+            <svg class="icon icon--chevron-down-16" focusable="false" height="16" width="16" aria-hidden="true">
+                <use href="#icon-chevron-down-16"></use>
+            </svg>
+        </span>
+        <div class="combobox__listbox">
+            <div id="listbox-chips-combobox-1" class="combobox__options" role="listbox">
+                <div class="combobox__option" role="option">
+                    <span>Baseball</span>
+                </div>
+                <div class="combobox__option" role="option">
+                    <span>Hockey</span>
+                </div>
+                <div class="combobox__option" role="option">
+                    <span>Tennis</span>
+                </div>
+            </div>
+        </div>
+    </span>
+</div>
+    {% endhighlight %}
+
+    <h3>Chips Combobox in Disabled State</h3>
+    <p>When disabled, Chips Combobox becomes "frozen" and users are unable to interact with the module - search for, add, and remove chips.</p>
+    <p>To turn on this state, add <span class="highlight">aria-disabled="true"</span> to the outer element of the module. Additonally, you'll need to add the <span class="highlight">disabled</span> attribute to all the buttons inside as well as the <span class="highlight">input</span> element for combobox.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="chips-combobox" aria-disabled="true">
+                <div class="chips-combobox__items" role="group" aria-describedby="chips-combobox-1-heading">
+                    <span class="chip">
+                        <span id="chip-interactive-1-1-text" class="chip__text">Football</span>
+                        <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-1-text">
+                            <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                {% include symbol.html name="close-12" %}
+                            </svg>
+                        </button>
+                    </span>
+                    <span class="chip">
+                        <span id="chip-interactive-1-2-text" class="chip__text">Basketball</span>
+                        <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-2-text">
```

Comment:
Looks like these are missing the disabled attribute as I can still focus on them:

<img width="367" alt="Screenshot 2023-12-18 at 11 47 35 AM" src="https://github.com/eBay/skin/assets/38065/e1fd603d-efe3-475f-a534-4b9f1fb61e4f">

Reply:
I'm able to focus on the textbox portion too.